commcare-cloud/commcare_cloud/commcare_cloud.py

```diff
@@ -516,6 +510,36 @@ def run_fab(args, unknown_args):
             puts(colored.red(u"✗ Fab failed with status code {}".format(exit_code)))
 
 
+class Ssh(object):
+    command = 'ssh'
+    help = "Connect to a remote host with ssh"
+
+    @classmethod
+    def make_parser(cls, parser):
+        cls.parser = parser
+        parser.add_argument("group",
```

Comment:
It might be worth mentioning that you can also name a server by "name" (since technically "named" servers are just putting the IP or DNS name under a group that contains just it), so `commcare-cloud softlayer ssh django0` and `commcare-cloud softlayer ssh webworkers:0` will incidentally both do the same thing. Even though your help string here is technically correct (`django0` is also a server "group"), I think it's worth calling out individually.

Nitpick: Also, since in the end this arg is supposed to signify a single server + username (which defaults via ssh/mosh to yours) and not a group of servers, I might call this arg `remote` or `server` or `remote_spec` or `server_spec` or something like that.

Not blocking, fine to do after if you like (or tell me to do it 😄)

Reply:
Thanks for the notes. I'll do this in a separate PR.